utils/RunONNXModel.py

```diff
@@ -736,8 +780,17 @@ def __init__(self, model_file=None, **kwargs):
                 args.load_model = args.cache_model
             args.cache_model = None
 
-        # Load the onnx model.
-        if args.model and args.model.endswith(".onnx"):
+        # Prepare onnx model for verifying output with onnxruntime
+        if (
+            not args.use_onnxmlir
+            and args.model
+            and args.model.endswith(".onnx")
+            and args.verify
+            and args.verify == "onnxruntime"
+            and args.verify_all_ops
+        ):
+            import onnx
```

Comment:
Should we have a `try` for the onnx import? Just to give a friendlier error message